subsys/caf/modules/Kconfig

```diff
@@ -17,3 +17,4 @@ rsource "Kconfig.power_manager"
 rsource "Kconfig.sensor_data_aggregator"
 rsource "Kconfig.sensor_manager"
 rsource "Kconfig.settings_loader"
+rsource "Kconfig.shell"
```

Comment:
Similarly, `Kconfig.caf_shell`

I'm not sure we're actually at risk of naming conflict, but it seems wise to just be explicit anyways

At minimum it'll make navigating to these source files easier